wallet-new/README.md

```diff
@@ -200,6 +222,7 @@ using environment variables as follows:
 ```
 LANG=en_GB.UTF-8 LC_ALL=en_GB.UTF-8 stack exec -- ...
 ```
+<<<<<<< HEAD
```

Comment:
This is useful to include instead, we should maintain in my opinion this section about "API returns 415 unsupported ..."